api/Element.json

```diff
@@ -6627,7 +6627,8 @@
                 "version_added": "37"
               },
               {
-                "version_added": true,
+                "version_added": "22",
```

Comment:
This matches https://caniuse.com/pointerlock, except caniuse.com doesn't capture the fact that Chrome 37 had both the prefixed and unprefixed API. (Confirmed on BrowserStack just now.)